specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2661 @@
+      "properties": {
+        "nextLink": {
+          "description": "The URI to fetch the next page of Cache SKUs.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of SKUs available for the subscription.",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ResourceSku"
+          },
+          "type": "array"
+        }
+      }
+    },
+    "NamespaceJunction": {
+      "description": "A namespace junction.",
+      "type": "object",
+      "properties": {
+        "namespacePath": {
+          "description": "Namespace path on a Cache for a Storage Target.",
+          "type": "string"
+        },
+        "targetPath": {
+          "description": "Path in Storage Target to which namespacePath points.",
+          "type": "string"
+        },
+        "nfsExport": {
+          "description": "NFS export where targetPath exists.",
+          "type": "string"
+        }
+      }
+    },
+    "StorageTargetsResult": {
+      "description": "A list of Storage Targets.",
+      "properties": {
+        "nextLink": {
+          "description": "The URI to fetch the next page of Storage Targets.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of Storage Targets defined for the Cache.",
+          "items": {
+            "$ref": "#/definitions/StorageTarget"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "URLString": {
+      "description": "A fully qualified URL.",
+      "type": "string"
+    },
+    "UsageModel": {
+      "description": "A usage model.",
+      "properties": {
+        "display": {
+          "description": "Localized information describing this usage model.",
+          "type": "object",
+          "properties": {
+            "description": {
+              "description": "String to display for this usage model.",
+              "type": "string"
+            }
+          }
+        },
+        "modelName": {
+          "description": "Non-localized keyword name for this usage model.",
+          "type": "string"
+        },
+        "targetType": {
+          "description": "The type of Storage Target to which this model is applicable (only nfs3 as of this version).",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "UsageModelsResult": {
+      "description": "A list of Cache usage models.",
+      "properties": {
+        "nextLink": {
+          "description": "The URI to fetch the next page of Cache usage models.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of usage models available for the subscription.",
+          "items": {
+            "$ref": "#/definitions/UsageModel"
+          },
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "amlFilesystem": {
```

Comment:
Need a couple more read-only properties here.
- MGS network address
- recommended mount command

Reply:
Can you provide some detail on these new fields?
mainly a field name,  description, type, and an example to put in the examples.

Reply:
With maintenance status left with only currentSoftwareVersion, should I just delete maintenance status completely?
I would then move currentSoftwareVersion and lustreVersion out to the main amlFilesystem object definition.